tailwind.config.js

```diff
@@ -55,5 +55,17 @@ export default {
       current: 'currentcolor',
       transparent: 'transparent',
     },
+    borderRadius: {
+      'none': '0',
+      'sm': '2px',
+      'DEFAULT': 'var(--border-radius)', // 4px
+      'md': 'var(--border-radius-medium)', // 6px
+      'lg': '8px',
+      'xl': '12px',
+      '2xl': '16px',
+      '3xl': '24px',
+      'circle': 'var(--border-radius-circle)', // 50%
+      'full': '9999px', // original tailwind value
```

Comment:
The "full" seems interesting (hmm, strange), maybe we should never use it or introduce it.

Reply:
I guess we could change `full` to `var(--border-radius-circle)` and change the uses of the circle.